code/game/gamemodes/steal_items.dm

```diff
@@ -67,31 +67,32 @@
 /datum/theft_objective/traitor/hand_tele
 	name = "a hand teleporter"
 	typepath = /obj/item/weapon/hand_tele
-	protected_jobs = list("Captain")
+	protected_jobs = list("Captain", "Research Director")
 
 /datum/theft_objective/traitor/rcd
 	name = "an RCD"
 	typepath = /obj/item/device/rcd/matter/engineering
-	protected_jobs = list("Chief Engineer")
+	protected_jobs = list("Chief Engineer", "Atmospherics Technician", "Station Engineer")
 
 /datum/theft_objective/traitor/rpd
 	name = "an RPD"
 	typepath = /obj/item/device/rcd/rpd
-	protected_jobs = list("Chief Engineer", "Atmospherics Technician")
+	protected_jobs = list("Chief Engineer", "Atmospherics Technician", "Station Engineer")//engineers have atmos access.
 
 /datum/theft_objective/traitor/jetpack
 	name = "a jetpack"
 	typepath = /obj/item/weapon/tank/jetpack
+	protected_jobs = list("Chief Engineer", "Captain", "Trader", "Atmospherics Technician", "Station Engineer", "Shaft Miner")
 
 /datum/theft_objective/traitor/cap_jumpsuit
 	name = "the captain's jumpsuit"
 	typepath = /obj/item/clothing/under/rank/captain
-	protected_jobs = list("Captain")
+	protected_jobs = list("Captain", "Head of Personnel")
 
 /datum/theft_objective/traitor/ai
 	name = "a functional AI"
 	typepath = /obj/item/device/aicard
-
+	protected_jobs = list("Captain", "Research Director", "Head of Personnel")
 
 /datum/theft_objective/traitor/magboots
 	name = "a pair of advanced magboots"
```

Comment:
If you're restricting the captain from jumpsuits, you'll probably want to do it from all the head items.